docs/scripts/reference.py

```diff
@@ -585,29 +586,29 @@ def _list_urls(urlpatterns: Sequence[Union[URLPattern, URLResolver]], prefix="")
     return urls
 
 
-def _format_tag_signature(tag_spec: TagSpec) -> str:
+def _format_tag_signature(node_cls: BaseNode) -> str:
```

Comment:
Before, the API reference for template tags was created from the `TagSpec` objects that were atttached to the template tag definitions. Now the same information is held directly by the `Node` classes